sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -238,7 +240,8 @@ case class AlterTableAddColumnsCommand(
         // TextFileFormat only default to one column "value"
         // Hive type is already considered as hive serde table, so the logic will not
         // come in here.
-        case _: JsonFileFormat | _: CSVDataSourceV2 | _: ParquetFileFormat | _: OrcDataSourceV2 =>
+        case _: JsonFileFormat | _: CSVFileFormat | _: ParquetFileFormat =>
```

Comment:
We should handle the V1 here as well, which was ignore in the previous migration PRs.

Reply:
Could you make this another small PR since this is Parquet migration PR?

Reply:
Got it, I will create another small PR with tests.